### ISSUE TITLE
"app" property in user response

### ISSUE THREAD

**Issue by paulyoung** [the-grid/apidocs#29]
This is present in the response when making a `GET` request to `https://passport.thegrid.io/api/user`, but isn't documented or shown in the schema.

Unless I'm mistaken it's the UUID of the app used to make the request.

It's not present when making a request to `https://passport.thegrid.io/api/user/{id}`, so can I assume that it's optional?

**Comment by trustmaster** [comment]
Yes, that's optional and comes from Passport.js. It is not linked with The Grid data entities.
